package.json

```diff
@@ -15,6 +15,7 @@
     "@material/menu": "^0.4.3",
     "@material/textfield": "^0.3.3",
     "@material/toolbar": "^0.4.4",
+    "@material/typography": "^0.3.0",
```

Comment:
for this change, when you ran `yarn add @material/typography` it should have also changed the `yarn.lock` file, we need to get those changes too to ensure we always install the same version.

Reply:
alice said "actually, you know what, you don't have to worry about that"